### ISSUE TITLE
Microsoft Edge: Transparent header in 5.x

### ISSUE THREAD

**Issue by lukas-koschmieder** [squidfunk/mkdocs-material#1728]
## Description

Microsoft Edge displays a transparent header in mkdocs-material 5.x. In the previous 4.x releases, Microsoft Edge used to display the background color correctly as defined in `theme['palette']['primary']`.

I've noticed that if you inspect the corresponding `md-header` DOM element, it will show you that the CSS attribures `color`/`background-color` are set correctly to `var(--md-primary-fg-color)`/`var(--md-primary-bg-color)`. So my guess is that there is something wrong with the fade-in animation; browser-specific instructions my be needed.

### Expected behavior

Edge renders the header background color as defined in `theme['palette']['primary']`

### Actual behavior

Edge renders a transparent header background

### Steps to reproduce the bug

1. Serve a mkdocs documentation with the mkdocs-material enabled
2. Open documentation in Edge

### Package versions

* Python: 3.7.3
* MkDocs: 1.1.2
* Material: 5.2.1

### Project configuration

``` yaml
theme:
  name: material
  language: en
  features:
    - instant
  palette:
    scheme: default
    primary: grey
    accent: blue
  font:
    text: Roboto
    code: Roboto Mono
  logo: img/logo.png
```

### System information

* OS: Windows 10
* Browser: Edge 44

**Comment by squidfunk** [comment]
I can't debug Edge, as I'm not on Windows (using macOS). Also, I've never heard of Edge 44. Could I ask you to try to debug the problem yourself?

**Comment by coliff** [comment]
I reproduced this issue too - and the text on the homepage is white (hidden) too:

![image](https://user-images.githubusercontent.com/1212885/82818618-f55fa780-9ed9-11ea-8f38-f1315e0d96ca.png)

Should really get this fixed as [Edge Legacy has approx 2% global marketshare](https://caniuse.com/usage-table)

**Comment by squidfunk** [comment]
As I said, I'm hoping for some help from somebody using Windows. It looks like it's related to CSS variables, which is strange, as [caniuse](https://caniuse.com/#feat=css-variables) says Edge 16+ supports them.

**Comment by coliff** [comment]
Changing the `--md-primary-fg-color` variable to `hsl(231, 48%, 48%)` fixes it.
Or just removing the `deg` part of the first value works too.. e.g. `hsla(231, 48%, 48%, 1)`

![image](https://user-images.githubusercontent.com/1212885/82823533-b97d1000-9ee2-11ea-92f5-cd2f9e7d41c8.png)

**Comment by lukas-koschmieder** [comment]
Thank you! This `deg` part seems to be generated by a SASS function `hex2hsl` in [_colors.scss](https://github.com/squidfunk/mkdocs-material/blob/master/src/assets/stylesheets/base/_colors.scss).
Do you have an idea how to fix it?

**Comment by squidfunk** [comment]
That is weird. Edge [should support `deg`](https://developer.mozilla.org/en-US/docs/Web/CSS/angle), but I guess since the grammar of the first value of `hsla()` is `<alpha> | <number>`, and `<number>` is implicitly converted to an `<angle>` value with the canonical `deg` unit, we could drop `deg` from all `hsla()` functions. It seems, that `deg` was added in [Color Level 4](https://www.w3.org/TR/css-color-4/#the-hsl-notation), as [Color Level 3](https://www.w3.org/TR/2018/REC-css-color-3-20180619/#hsla-color) doesn't mention it.

**Comment by squidfunk** [comment]
Fixed in 24b185a7.

**Comment by squidfunk** [comment]
Released as part of 5.2.2
